src/ui/public/vis/editors/default/agg_add.html

```diff
@@ -2,6 +2,7 @@
   <label>Select {{ groupName }} type</label>
   <ul class="form-group list-group list-group-menu">
     <li
+      tabindex="0"
```

Comment:
`tabindex` unfortunately also needs the `id` attribute on this element to work properly. We documented this [in the styleguide](https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md#use-tabindex-to-make-elements-tabbable)